app/models/tweet_prep.rb

```diff
@@ -0,0 +1,29 @@
+class TweetPrep
+  def self.bounty_hashtags(bounty)
+    QueryMarks.new.legible_mark_vector(bounty.mark_vector).sort_by{|a, b| -b}.map{|a, b| a}.take(4)
+  end
+
+  def self.worthy_bounties(n)
+    bounty_suggestion_fraction = 0.5
+    sorted_top_bountys = bounties_from_top_products.select{|a| a.earnable_coins_cache}.sort_by{|a| -a.earnable_coins_cache}
+    top_bountys = sorted_top_bountys.take(top_bountys.count * bounty_suggestion_fraction)
+    top_bountys.sample(n).uniq
+  end
+
+  def self.bounties_from_top_products
+    top_products_by_activity.map{|a| a.tasks}.flatten.select{|a| a.state == "open"}
+  end
+
+  def top_products_by_activity
+    ProductStats.top_products_by_activity(limit: 12).select{|a, b| a != "meta"}.map{|a, b| Product.find_by(slug: a)}
```

Comment:
Unused block argument - `b`. If it's necessary, use `_` or `_b` as an argument name to indicate that it won't be used.<br>Line is too long. [116/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.